contrib/python/pypodman/pypodman/lib/parser_actions.py

```diff
@@ -94,21 +91,40 @@ def __init__(self,
             help=help,
             metavar=metavar)
 
-    def __call__(self, parser, namespace, values, option_string=None):
-        """Convert and Validate input."""
-        items = getattr(namespace, self.dest, None) or []
-        items = copy.copy(items)
+        if hasattr(signal, "Signals"):
```

Comment:
This looks like a python2-or-3 handler. I thought pypodman required python3?

Reply:
You are correct those styles of checks are most scene 3.x vs. 2.x.  But I need this one for < 3.5 support. The Signals enum was introduced in python 3.5, I have been base-lining code for python 3.4 for RHEL SCL support.